ci/ci.el6/integration_test/multibox/ind-steps/step-box/preconfigure.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+run_cmd "iptables -F"
```

Comment:
You are correct, its a copy/paste miss, where in centos 7 we use systemctl to disable, which can be done before booting the vm, fixed this.